sdk/logs/src/main/java/io/opentelemetry/sdk/logs/data/LogDataBuilder.java

```diff
@@ -86,6 +86,7 @@ public LogDataBuilder setSeverityText(String severityText) {
   }
 
   /** Set the name. */
```

Comment:
might be nice to document the deprecation in javadoc and explain that it will be removed with no replacement.